api/mesh/v1alpha1/dataplane.proto

```diff
@@ -183,8 +183,22 @@ message Dataplane {
       // specific data plane proxy of a service (for example: `version=v2`) the
       // better practice is to use TrafficRoute.
       map<string, string> tags = 5;
+
+      // BackendRef is a way to target MeshService.
+      // Experimental. Do not use on production yet.
```

Comment:
Should mention that if both tags and backendRef is set. backendRef takes over.

Reply:
that should not be allowed. Added validation